lib/plugins/wallet/bitgo/bitgo.js

```diff
@@ -94,14 +97,28 @@ function newAddress (account, info) {
 }
 
 function getStatus (account, toAddress, requested, cryptoCode) {
-  const bitgo = buildBitgo(account)
   return checkCryptoCode(cryptoCode)
-    .then(() => bitgo.blockchain().getAddress({ address: getLegacyAddress(toAddress, cryptoCode) }))
-    .then(rec => {
-      if (rec.balance === 0) return { status: 'notSeen' }
-      if (requested.gt(rec.balance)) return { status: 'insufficientFunds' }
-      if (requested.gt(rec.confirmedBalance)) return { status: 'authorized' }
-      return { status: 'confirmed' }
+    .then(() => getWallet(account, cryptoCode))
+    .then(wallet => wallet.transfers({ type: 'receive', address: toAddress }))
+    .then(({ transfers }) => {
+      const filterConfirmed = _.filter(it =>
+        it.state === 'confirmed' && it.type === 'receive'
+      )
+      const filterUnconfirmed = _.filter(it =>
```

Comment:
Change this to ``filterPending`` and have it check if state is either unconfirmed or confirmed. Then we won't have to add ``confirmed`` records later on.